<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSPHWaterCube</class>
 <widget class="QWidget" name="WidgetSPHWaterCube">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Gravity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="gravity">
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>9.810000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Black Hole</string>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>150</width>
        <height>18</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="btnUpdate">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Move object:</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Sphere</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Box</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Blackhole</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>SPH Method:</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>h reduction</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rest density</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed of sound</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_h_reduction">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.950000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_rest_density">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
      <property name="value">
       <double>0.002000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_c">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>1500.000000000000000</double>
      </property>
      <property name="value">
       <double>300.000000000000000</double>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Fully Compressible Fluid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Weakly Compressible Fluid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Iterative Weakly Compressible Fluid</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kinematic Viscosity</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_kinematic_viscosity">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="singleStep">
       <double>0.000010000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stiffness</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBox_k">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDefaultParameters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Default Parameters</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
